nx/lib/nx.ex

```diff
@@ -7046,6 +7046,123 @@ defmodule Nx do
     impl!(tensor).sort(tensor, tensor, axis: axis, comparator: comparator)
   end
 
+  @doc """
+  Calculates the QR decomposition of an 2-D tensor with shape `{M, N}` via the Householder method.
```

Comment:
Just a nit, some backends may not use the Householder method, so you don't need to mention it!Also "an" should be "a" :)